packages/extension/components/RelatedIssueItem.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <a :href="issue.url" class="my-1 truncate block flex-row gap-1 gap-2">
+      <svg v-if="issue.state === 'open'" class="octicon octicon-issue-opened open" title="Open" aria-label="Open issue"
+           viewBox="0 0 16 16" version="1.1" width="16" height="16" role="img">
+        <path d="M8 9.5a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3Z"></path>
+        <path d="M8 0a8 8 0 1 1 0 16A8 8 0 0 1 8 0ZM1.5 8a6.5 6.5 0 1 0 13 0 6.5 6.5 0 0 0-13 0Z"></path>
+      </svg>
+
+      <svg v-else class="octicon octicon-issue-closed closed" title="Closed" aria-label="Closed issue"
+           viewBox="0 0 16 16" version="1.1" width="16" height="16" role="img" style="color: #a371f7;">
+        <path
+            d="M11.28 6.78a.75.75 0 0 0-1.06-1.06L7.25 8.69 5.78 7.22a.75.75 0 0 0-1.06 1.06l2 2a.75.75 0 0 0 1.06 0l3.5-3.5Z"></path>
+        <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0Zm-1.5 0a6.5 6.5 0 1 0-13 0 6.5 6.5 0 0 0 13 0Z"></path>
+      </svg>
+
+      <RelatedIssueTitle class="ml-2 !text-white" :text="issue.title"></RelatedIssueTitle>
+      <br>
+    </a>
+    <a class="ml-4 hover:underline focus:underline" :href="issue.url.split('/issues/')[0]">{{
```

Comment:
this was 
```html
<a class="ml-4 hover:underline focus:underline" :href="`https://github.com/${issue.owner}/${issue.repository}`" >
``` 
before. But I thought we might want to be independent of the server